models/__init__.py

```diff
@@ -0,0 +1,38 @@
+# coding=utf-8
```

Comment:
`__init__.py` should not contain code, but rather just imports (if needed). If submodules are not expected, then this can be a one file, a module, named `modules.py`.